unit_test/test_spdm_crypt/test_spdm_crypt.c

```diff
@@ -35,6 +35,110 @@ uint8_t m_libspdm_subject_alt_name_buffer3[] = {
 uint8_t m_libspdm_dmtf_oid[] = { 0x2B, 0x06, 0x01, 0x4,  0x01,
                                  0x83, 0x1C, 0x82, 0x12, 0x01 };
 
+uint8_t m_cert_buffer[] = {
```

Comment:
1. Add instruction for `m_cert_buffer` generated from which file.
2. Add the following link to make the `m_cert_buffer` easy to understand: https://lapo.it/asn1js/